src/coreclr/jit/emitxarch.h

```diff
@@ -65,11 +65,14 @@ BYTE* emitOutputRRR(BYTE* dst, instrDesc* id);
 
 BYTE* emitOutputLJ(insGroup* ig, BYTE* dst, instrDesc* id);
 
+unsigned emitOutputSIMDPrefixIfNeeded(instruction ins, BYTE* dst, code_t& code);
```

Comment:
A personal nit: I've been trying to use `Simd` rather than `SIMD` in new code. Likewise `Avx` rather than `AVX` and `Vex` rather than `EVEX`, etc.

This "better" matches the managed .NET naming convention and makes it simpler to map concepts between managed/native. I think it likewise makes the names a bit easier to read.

We aren't super consistent overall, however, and the casing often depends on when it was added and who added it.

Reply:
I have changed All SIMD,AVX512,EVEX to Simd, Avx512, Evex. I left all pre-existing ones the way they were.